frontend/src/api/index.js

```diff
@@ -88,8 +95,21 @@ export const setApplicationRating = (applicationId, rating) =>
     },
   });
 
+export const getSelfInfo = () => authenticatedRequest({ path: "/user" });
+
 export const getApplicationAnswers = (applicationId) =>
   authenticatedRequest({ path: `/application/${applicationId}/answers` });
 
 export const getApplicationRatings = (applicationId) =>
   authenticatedRequest({ path: `/application/${applicationId}/ratings` });
+
+export const submitAnswer = (application_id, question_id, description) =>
+  authenticatedRequest({
+    method: "POST",
+    path: `/application/answer/`,
```

Comment:
Can use a normal string here instead of template literal if no params (although not sure why you moved application_id to json body instead of url)